### DISCUSSION TITLE
[Question] Is the steps hidden when the test pass?

### DISCUSSION THREAD

**Discussion by apis3445** [ortoniKC/ortoni-report#27] [Q&A] [answered]
Hello! I tried the report, and when the test was passed, I couldn't see the steps; it only showed when the test failed.
![image](https://github.com/user-attachments/assets/1d3388da-9420-4cb8-982f-6a344e752727)

![image](https://github.com/user-attachments/assets/086c02fd-8e67-4ba5-aa97-1da030621517)

**Comment by ortoniKC** [comment]
Seems like there is an issue with the css loading, 
The default behaviour is when the test fails it will open the steps section and when the test passes it will collapse the step section, you can click on the steps arrow to view the steps.

**Comment by apis3445** [comment]
I clicked and It is the same is not loading any steps collapse and uncollapse is empty

**Comment by ortoniKC** [comment]
Is it API related? 
And you're using apiContext not the page?

**Comment by apis3445** [comment]
Hello! sorry for the delay is not with API context, but maybe because is a function like this

    async addStep(stepDescription: string, stepFunction: any): Promise<any> {
        return await test.step(stepDescription, stepFunction);
    }

**Reply by ortoniKC** [reply]
It is possible to provide a sample repo?

**Reply by apis3445** [reply]
Although includes a lot of samples is this: https://github.com/apis3445/Playwright

**Comment by ortoniKC** [comment, marked as answer]
Steps are shown by default now.
Closing this now. 
Let me know if you're still facing the same issue.

**Comment by apis3445** [comment]
is working correctly thank you
